components/Habit.tsx

```diff
@@ -62,17 +69,32 @@ type HabitProps = {
   streak: number;
   habit: THabit;
   entries: Entry[];
+  onAddHabitEntry: () => void;
+  onRemoveHabitEntry: (entryId: string) => void;
 };
 
-const Habit = ({ entries, habit, streak }: HabitProps) => {
+const Habit = ({
+  entries,
+  habit,
+  streak,
+  onAddHabitEntry,
+  onRemoveHabitEntry,
+}: HabitProps) => {
   const { t } = useTranslation(['common', 'habit']);
 
   const todayEntries = getEntriesForToday(entries);
+  const isComplete = () => todayEntries.length >= 1;
```

Comment:
[non-blocking] this doesn't need to be a function, since it is always used - switch over to a constant ;)